crud.py

```diff
@@ -123,9 +110,9 @@ async def get_pay_links(wallet_ids: Union[str, List[str]]) -> List[PayLink]:
 
 
 async def update_pay_link(link_id: str, **kwargs) -> Optional[PayLink]:
-    if "lnaddress" in kwargs:
-        await check_lnaddress_format(kwargs["lnaddress"])
-        await check_lnaddress_update(kwargs["lnaddress"], link_id)
+    if len(kwargs["username"]) > 0:
```

Comment:
`username` is though always present

    with an empty string when there's no lnaddress bound to the pay link
    with `username `part of the lnaddress with a lnaddress is bind to the pay link